mne/io/cnt/_utils.py

```diff
@@ -0,0 +1,95 @@
+# Author: Joan Massich <mailsik@gmail.com>
+#
+# License: BSD (3-clause)
+
+from struct import Struct
+from collections import namedtuple
+from math import modf
+from datetime import datetime
+
+from ...utils import warn
+
+
+def _read_TEEG(f, teeg_offset):
```

Comment:
TEEG -> teeg in function name

Reply:
true, even if its an accronim we have no capitals in any function name.